code/obj/critter/pets_small_animals.dm

```diff
@@ -901,9 +902,21 @@ var/list/shiba_names = list("Maru", "Coco", "Foxtrot", "Nectarine", "Moose", "Pe
 		if (!src.alive || src.sleeping || !text)
 			return
 		var/m_id = (lang_id == "english" || lang_id == "") ? 1 : 2
-		if (prob(learn_words_chance))
+		if (M.singing)
+			if (M.singing == "bad" || M.singing == "loud")
+				spawn(3)
```

Comment:
why spawn

Reply:
don't want the bird reaction text to appear before the singing text